.github/workflows/beaker.yml

```diff
@@ -162,6 +162,11 @@ jobs:
     name: "${{ matrix.name }}"
     steps:
       - uses: actions/checkout@v4
+      - name: Setup podman
```

Comment:
Can you move this until after `install additional packages`? While the ubuntu-24.04 image has it, moving it allows working around it when a user does pass `acceptance_runs_on` as a parameter.

Reply:
Sure, done!